pkg/types/installconfig.go

```diff
@@ -85,6 +90,13 @@ type InstallConfig struct {
 	// +optional
 	AdditionalTrustBundle string `json:"additionalTrustBundle,omitempty"`
 
+	// AdditionalTrustBundlePolicy defines conditional policy to add AdditionalTrustBundle
+	// to the nodes' trusted certificate store. If omitted, "Default" is considered.
+	// The field can be set to following specified values.
+	// "Default" : uses AdditionalTrustBundle when http/https proxy is configured.
```

Comment:
I don't think `Default` is a good value here. Perhaps something like `Proxy` or `ProxyOnly` and then we declare that this value is the default.

Reply:
Yes makes sense. Updated the _Default_ to _ProxyOnly_. Thanks for the review.